### ISSUE TITLE
How to summary results as "0" or "1"  or ("2" for carry two same genes)? Thanks!

### ISSUE THREAD

**Issue by lancetxiao** [tseemann/abricate#73]
<img width="967" alt="image" src="https://user-images.githubusercontent.com/21260389/49379270-ad7b1c80-f749-11e8-960c-15d6f175a14e.png">
How about If one strain carry two or more same genes ?
<img width="286" alt="image" src="https://user-images.githubusercontent.com/21260389/49379410-03e85b00-f74a-11e8-9773-29c18e1ffbd7.png">

**Comment by tseemann** [comment]
I think you are asking for the `abricate --summary` to have the number 0, 1, 2 etc instead of the % values?

This could be made an option.

**Comment by lancetxiao** [comment]
YES, That is what I means. Use the "abricate --summary" to have the number 0, 1, 2 etc instead of the % values. We can set the %value in "abricate --mincov" before. I think the  “number 0, 1, 2”   will be more helpful in subsequent analysis for people who are not good at command-line operations, LIKE ME. Thank you again!

**Comment by ireneortega** [comment]
I agree with @lancetxiao %coverage does not provide any useful information in the matrix for me. With --mincov and --minid options selected before, you already know that values. If it is a **presence/absence genes matrix**, I would use 1 and 0. In fact, I am changing values to 0 and 1.

**Comment by genomesandMGEs** [comment]
Do you guys have any idea on how to convert this to number of occurrences of each genes? I guess that more than just changing to 0 and 1, it would be good to replace the number of coverage values by the total number of occurrence of each hit.
